Add cfp details

rjw1 opened the pull request:
preview of CFP page changes

rjw1 committed:
[LDN-2019] Make submission links more noticeable

Make them a H2 and centre them.

rjw1 committed:
[LDN-2019] Add details about what speakers get

rjw1 committed:
[LDN-2019] Add talk submission help section

chriscoffee commented:
Looks like this already got merged upstream? This was for the netlify link right?

Can we close this now?

rjw1 commented:
did it. I've not sent this one yet. I need to make changes yet.

MikeRosTX committed:
Merge pull request #6693 from notsureifkevin/master

[bna-2019] removed CFP link, updated program/speakers link

Commit:
[ZRH-2019] Add AdNovum as Bronze sponsor

bridgetkromhout committed:
Merge pull request #6695 from Roderick-Jonsson/master

[ZRH-2019] Add AdNovum as Bronze sponsor

Commit:
Updates organizers list: Removes André Déo and adds Christiano Linuxmen; adds organizers bios.

matiasfrndz committed:
[ZRH-2019] Program update

MikeRosTX committed:
Merge pull request #6696 from devopsdays-campinas/CAMP-2019

[CAMP-2019] Updates organizers list

MikeRosTX committed:
Merge pull request #6697 from matiasfrndz/zrh-2019-second-program-update

[ZRH-2019] Program update

Commit:
[BER-2019] Add talent.io as sponsor

Commit:
[BER-2019] Also commit logo and file :/

kmugrage committed:
update stem sponsorship

kmugrage committed:
update text

MikeRosTX committed:
Merge pull request #6698 from range/master

[BER-2019] Add talent.io as sponsor

kmugrage committed:
Merge pull request #6699 from kmugrage/stem-sponsor

[SEA-2019] update stem sponsorship

kmugrage committed:
[BLN-2019] refactor to support talentio logo

bridgetkromhout committed:
Merge pull request #6700 from kmugrage/master

[BLN-2019] refactor to support talentio logo

PaulusTM committed:
[AMS-2019] Add Fullstaq as foodtruck sponsor

Signed-off-by: Daniel Paulus <d.paulus@gmail.com>

kmugrage committed:
Merge pull request #6701 from dpnl87/sponsor/fullstaq

[AMS-2019] Add Fullstaq as foodtruck sponsor

delgod committed:
[Kiev-2019] add ignite talks

delgod committed:
[Kiev-2019] add Yurii Rochniak

delgod committed:
[Kiev-2019] update Victor Voloshchak bio

MikeRosTX committed:
Merge pull request #6702 from delgod/add-kiev

[KIEV-2019] add organizer & ignite talks

PaulusTM committed:
[AMS-2019] Add VMware as gold

Signed-off-by: Daniel Paulus <d.paulus@gmail.com>

MikeRosTX committed:
Merge pull request #6703 from dpnl87/sponsor/vmware

[AMS-2019] Add VMware as gold

Commit:
[VAN-2019] Swap Nova and Longworth

Commit:
[VAN-2019] Speaker cancellation

MikeRosTX committed:
Merge pull request #6704 from delgod/add-kiev

[Kiev-2019] update Victor Voloshchak bio

MikeRosTX committed:
Merge pull request #6705 from tylermauthe/VAN-2019-speaker-fixes

[VAN-2019] Speaker fixes

Commit:
speakers photo updated.

halil9 committed:
Merge branch 'master' into master

halil9 committed:
DevopsDays Istanbul 19 (#6706)

[IST-2019] update speaker photos (#6706)

eduardopiairo committed:
Merge branch 'master' into pt2019-program

michaelthomasclark committed:
Update event location and organizer info

WhatsChrisDoing committed:
Blended Perspectives now a platinum sponsor

eduardopiairo committed:
program, mirco hering abstract, edp logo resize

streichsbaer committed:
[JKT-19] Add sponsors and update program

MikeRosTX committed:
Merge pull request #6707 from WhatsChrisDoing/sponsor-update

[TOR-2019] sponsor update

MikeRosTX committed:
Merge pull request #6708 from devopsdaysportugal/pt2019-keynote1-abstract

[PT2019] keynote1 abstract

Commit:
speakers photo updated.

Commit:
Merge branch 'master' of https://github.com/halil9/devopsdays-web

Commit:
speaker photo updated.

cobusbernard committed:
[CPT-2019] Adding link for Cape Town prospectus (#6711)

* Adding link for Cape Town prospectus

* Adding back the delete tag

* Removing the 2nd please

halil9 committed:
DevopsDays Istanbul 19  (#6710)

[IST-2019] update speaker bios & photos (#6710)

skryukova committed:
[HOU-2019] Added speaker

MikeRosTX committed:
Merge pull request #6712 from skryukova/master

[HOU-2019] Added speaker

bridgetkromhout committed:
Merge pull request #6709 from streichsbaer/master

[JKT-19] Add sponsors and update program

eduardopiairo committed:
gfi as bronze sponsor

MikeRosTX committed:
Merge pull request #6713 from devopsdaysportugal/pt2019-sponsor-gfi

[PT2019] gfi as bronze sponsor

jaumann committed:
[BMORE-2019] Fix collabnet logo

Commit:
[TPA-2019] - Adding Accenture as Sponsor - TY!

MikeRosTX committed:
Merge pull request #6714 from jaumann/bmore_update_cn_logo

[BMORE-2019] Fix collabnet logo

MikeRosTX committed:
Merge pull request #6715 from jmgress/2019-tampa

[TPA-2019] - Adding Accenture as Sponsor - TY!

Commit:
add registration link

MikeRosTX committed:
Merge pull request #6716 from jakubjakubik/master

POZ-2019 - add registration link

JasmineCrozier committed:
[CPH-2019] Add Nuuday

MikeRosTX committed:
Merge pull request #6717 from Praqma/add-nuuday

[CPH-2019] Add Nuuday

bushong1 committed:
[BMORE-2019] Add gcloud, cloudbolt; add/create steamnastics

MikeRosTX committed:
Merge pull request #6718 from devopsdaysbaltimore/bmore/sponsors

[BMORE-2019] Add sponsors gcloud, cloudbolt; add/create stemnastics

bushong1 committed:
[BMORE-2019] Create/add gigabyte_magazine

Commit:
Merge branch 'master' of github.com:devopsdays/devopsdays-web

Commit:
Updating nick herrig bio and picture

jeanmorais committed:
[FLN-2019] Adding Florianópolis in DevOpsDays events.

Commit:
Adding chris hunt to the program

MikeRosTX committed:
Merge pull request #6719 from devopsdaysbaltimore/bmore/gigabyte_magazine

[BMORE-2019] Create/add gigabyte_magazine

Commit:
speakers bio updated.

halil9 committed:
Merge branch 'master' into master

MikeRosTX committed:
Merge pull request #6721 from halil9/master

[IST-2019] Speaker bio and photo updates

Commit:
speaker program updated.

Commit:
Merge branch 'master' of https://github.com/halil9/devopsdays-web

bridgetkromhout committed:
Merge pull request #6722 from halil9/master

DevopsDays Istanbul 19

Commit:
Adding Russ Penning to the program

Commit:
Updating program

Commit:
Adding Emily Nakashima to the program

normalfaults committed:
[RDU-2019] added sponsor and updated propose deadline

[RDU-2019] added sponsor and updated propose deadline

jpadilla committed:
[HFD-2019] Update logo and welcome page

JasmineCrozier committed:
add free parking

MikeRosTX committed:
Merge pull request #6723 from jessiepuls/master

[DSM-2019] Updating Program

MikeRosTX committed:
Merge pull request #6724 from normalfaults/patch-25

[RDU-2019] added sponsor and updated propose deadline

MikeRosTX committed:
Merge pull request #6725 from jpadilla/hfd-2019

[HFD-2019] Update logo and welcome page

MikeRosTX committed:
Merge pull request #6726 from Praqma/add-parking

[CPH-2019] add free parking

Commit:
[DEN-2019] More sponsors and change to page

bridgetkromhout committed:
Merge pull request #6727 from endzyme/den-adding-sponsors

[DEN-2019] More sponsors and change to page

eduardopiairo committed:
fix mirco hering photo

DebbieGillespie committed:
[MSP-2019] Add oracle and keyva as silver sponsors

Although we already had several logos for Oracle, none of them matched
the logo the sponsor requested, so I added yet another entry.

kmugrage committed:
added drayage info

kmugrage committed:
added fedex info

kmugrage committed:
Merge pull request #6729 from kmugrage/sponsors

[SEA-2019] added drayage info

MikeRosTX committed:
Merge pull request #6730 from devopsdaysportugal/pt2019-mircophotofix

[PT2019] fix mirco hering photo

kmugrage committed:
Merge pull request #6728 from DebbieGillespie/add_oracle_keyva

[MSP-2019] Add oracle and keyva as silver sponsors

jeanmorais committed:
Removed deprecated label cfp_open

BGnoinski committed:
[victoria-2019] updated registration page, added placeholder and image for potential sponsor

bridgetkromhout committed:
Merge pull request #6731 from BGnoinski/2019-victoria-yyj

[victoria-2019] updated registration page, added placeholder and imag…

Commit:
[VAN-2019] Program fixes

- Add speaker profile for Nisan
- More details for evening event

Commit:
[VAN-2019] Add Nisan's tweeter profile

Commit:
speakers added.

Commit:
speakers photo updated.

Commit:
[ZRH-2019] Add Switch as bronze sponsor

apxltd committed:
[TYO2019] Update Organizers

scottsbaldwin committed:
[ATX-2019] Add more gold sponsors

* Avi Networks
* Signal Sciences

MikeRosTX committed:
Merge pull request #6720 from jeanmorais/fln-2019

[FLN-2019] WIP - Adding Florianópolis in DevOpsDays events

MikeRosTX committed:
Merge pull request #6734 from Roderick-Jonsson/master

[ZRH-2019] Add Switch as bronze sponsor

MikeRosTX committed:
Merge pull request #6732 from tylermauthe/VAN-2019-last-minute-program-fixes

[VAN-2019] Program fixes

MikeRosTX committed:
Merge pull request #6733 from halil9/master

[IST-2019] Adding and updating speaker details

bridgetkromhout committed:
Merge pull request #6736 from scottsbaldwin/atx/more-mar-sponsors

[ATX-2019] Add more gold sponsors

bridgetkromhout committed:
Merge pull request #6735 from apapadimoulis/patch-1

[TYO-2019] Update Organizers

Commit:
[PDX-2019] Added additional gold sponsors

WhatsChrisDoing committed:
Add SPS Commerce as gold and lanyard sponsor

Commit:
Added Jaqueline talk description

jpadilla committed:
[HFD-2019] Open CPFs

bridgetkromhout committed:
Merge pull request #6738 from DevOpsDaysPDX/master

[PDX-2019] Added additional gold sponsors

bridgetkromhout committed:
Merge pull request #6739 from WhatsChrisDoing/sponsor-update

[TOR-2019] Add SPS Commerce as gold and lanyard sponsor

bridgetkromhout committed:
Merge pull request #6740 from fike/add_speaker_description

[SP-2019] Added Jaqueline talk description

bridgetkromhout committed:
Merge pull request #6741 from jpadilla/hfd-2019

[HFD-2019] Open CPFs

eduardopiairo committed:
sponsorship program upgrade: number of tickets increase

JasmineCrozier committed:
[CPH-2019] Remove Felix and add Rasmus

bridgetkromhout committed:
Merge pull request #6742 from devopsdaysportugal/sponsorship-program-upgrade

[PT2019] sponsorship program upgrade: number of tickets increase

bridgetkromhout committed:
Merge pull request #6743 from Praqma/change-speaker

[CPH-2019] Remove Felix and add Rasmus

bridgetkromhout committed:
Temporary patch for Sharp issue

bridgetkromhout committed:
Merge pull request #6744 from devopsdays/sharp-temp-fix

Temporary patch for Sharp issue

Commit:
Replace Cloud Native London Logo

Cloud Native London have a better/new logo and Dom has asked for it to
be replaced.

Signed-off-by: Chris M <me@christophermills.co.uk>

Commit:
Added new sponsors.

* C6bank
* Indra
* Agility
* Instana

Commit:
speaker added.

delgod committed:
[Kiev-2019] update Julia Tkachova talk

bridgetkromhout committed:
Merge pull request #6747 from fike/last_sponsors

[SP-2019] last round of sponsors.

bridgetkromhout committed:
Merge pull request #6748 from halil9/master

DevopsDays Istanbul 19

bridgetkromhout committed:
Merge pull request #6749 from delgod/add-kiev

[Kiev-2019] update Julia Tkachova talk

bridgetkromhout committed:
Merge pull request #6746 from devopsdays-london/ldn-2019-community-change

[LDN-2019] Replace Cloud Native London Logo

scottsbaldwin committed:
[ATX-2019] Cyberark and Riverbed sponsors

* also updates to the welcome page links

Commit:
Added speakers and program

BenLubar committed:
[2019-Tokyo] Add sponsors

Update sponsors to match https://www.devopsdaystokyo.org/

bridgetkromhout committed:
Merge pull request #6750 from scottsbaldwin/atx/moar-mar-sponsors

[ATX-2019] Cyberark and Riverbed sponsors

Commit:
Adjusted naming of first talk

MikeRosTX committed:
Merge pull request #6751 from BenLubar/2019-tokyo-sponsors

[2019-Tokyo] Add sponsors

MikeRosTX committed:
Merge pull request #6752 from endzyme/den-program

[DEN-2019] Added speakers and program

tonybolzan committed:
[SM-2019] Logo Updated

MikeRosTX committed:
Merge pull request #6753 from devopsdays-sm/master

[SM-2019] Logo Updated

streichsbaer committed:
[JKT-19] Add sponsors and update program

MikeRosTX committed:
Merge pull request #6754 from streichsbaer/master

[JKT-19] Update program

streichsbaer committed:
[JKT-19] Update talk abstract

skryukova committed:
[HOU-2019] Added Oleg Atamanenko photo and social media info

skryukova committed:
[HOU-2019] Adjusted times for workshops, removed Istio workshop

skryukova committed:
[HOU-2019] Added new workshop and speaker Brad Dausses

MikeRosTX committed:
Merge pull request #6757 from skryukova/master

[HOU-2019] Adjustments to the program and speakers

MikeRosTX committed:
Merge pull request #6756 from streichsbaer/master

[JKT-19] Update talk abstract

WhatsChrisDoing committed:
Adjust Blended Perspectives logo

MikeRosTX committed:
Merge pull request #6758 from WhatsChrisDoing/sponsor-logo-updare

[TOR-2019] Adjust Blended Perspectives logo

mazek committed:
[POZ-2019] Adding first sponsors.

MikeRosTX committed:
Merge pull request #6759 from mazek/master

[POZ-2019] Adding first sponsors for Poznan 2019.

streichsbaer committed:
[JKT-19] Update speaker bio

streichsbaer committed:
[JKT-19] Update program

MikeRosTX committed:
Merge pull request #6760 from streichsbaer/master

[JKT-19] Update speaker bio

ghoneycutt committed:
[INDY-2019] Close CFP

bridgetkromhout committed:
Update 2019-indianapolis.yml

This is how to remove the propose link.

MikeRosTX committed:
Merge pull request #6761 from ghoneycutt/close_cfp

WIP - [INDY-2019] Close CFP

mazek committed:
General cleaning unsued and badly filled .yml files.

mazek committed:
[POZ-2019] Updated sponsors list.

MikeRosTX committed:
Merge pull request #6762 from mazek/master

[POZ-2019] General cleaning unsued and badly filled .yml files.

Commit:
Adding James Young

clumpidy committed:
[DC-2019] Adding myself to the list of organizers for Washington DC 2019

Commit:
Adding Pete Anderson

MikeRosTX committed:
Merge pull request #6763 from clumpidy/jake_update_organizers

[DC-2019] Adding myself to the list of organizers for Washington DC 2019

Commit:
Adding Jason Yee

ecology-chris committed:
added Poppy, Spencer, updated ACS's talk title. Added VMware

Commit:
Adding Dustin Tostenson and Skylar Watson

Commit:
Adding Stephen Wilson

MikeRosTX committed:
Merge pull request #6764 from cacorriere/schedule_update

[ATL-2019] schedule update + vmware

Commit:
Adding Martin Schildroth

Commit:
Adding talks to program

Commit:
Adding Kylee Tilley

Commit:
Updating program

Commit:
Merge branch 'master' of github.com:devopsdays/devopsdays-web

MikeRosTX committed:
Merge pull request #6765 from jessiepuls/master

[DSM-2019] Updating program

MikeRosTX committed:
Update location.md

MikeRosTX committed:
Merge pull request #6755 from adzuci/master

[BOS-2019] Update event location and organizer info

xavidop committed:
intial madrid 2020 events

skryukova committed:
[HOU-2019] Added social media info for Brad Dausses

xavidop committed:
misspell checked

bridgetkromhout committed:
Merge pull request #6767 from skryukova/master

[HOU-2019] Added social media info for Brad Dausses

bridgetkromhout committed:
Update contact.md

Your organizer team will be listed because of "list_organizers" - you don't need to manually add them here (and you should not, since that's a place that your data could get out of sync).

MikeRosTX committed:
Merge pull request #6768 from xavierportillaedo/master

[MAD-2020] Adding placeholder

Commit:
sponsors added.

bridgetkromhout committed:
Merge pull request #6769 from halil9/master

[IST-2019] Adding sponsors

bridgetkromhout committed:
Revert "Temporary patch for Sharp issue"

Commit:
[DEN-2019] Added Oracle Coffee Sponsorship

bridgetkromhout committed:
Merge pull request #6770 from devopsdays/revert-6744-sharp-temp-fix

Revert "Temporary patch for Sharp issue"

MikeRosTX committed:
Merge pull request #6771 from endzyme/den-sponsors

[DEN-2019] Added Oracle Coffee Sponsorship

WoodProgrammer committed:
DevOps days 2019 Istanbul sponsors img changed

bridgetkromhout committed:
Merge pull request #6772 from halil9/master

DevOps days 2019 Istanbul sponsors img changed

PaulusTM committed:
[AMS-2019] Remove the pre event from the sponsor levels

Signed-off-by: Daniel Paulus <d.paulus@gmail.com>

MikeRosTX committed:
Merge pull request #6773 from dpnl87/sponsor/level_update

[AMS-2019] Remove the pre event from the sponsor levels

Commit:
Sponsor Plans and New Speakers and Sponsor added

MikeRosTX committed:
Updating header information

Updating header information.

MikeRosTX committed:
Update Rudy-Gevaert.md

Translation of bio to Spanish.

MikeRosTX committed:
Merge pull request #6774 from devopscolombia/master

[BOG-2019] Added new Sponsors and Speakers

mazek committed:
[POZ-2019] Change in sponsors.

MikeRosTX committed:
Merge pull request #6775 from mazek/master

[POZ-2019] Change in sponsors.

Commit:
Changed program.

bridgetkromhout committed:
Merge pull request #6776 from fike/changed_program

[SP-2019] Changed program

toshywoshy committed:
[GHENT-2019] add mailchimp link

Signed-off-by: Toshaan Bharvani <toshaan@vantosh.com>

MikeRosTX committed:
Merge pull request #6777 from toshywoshy/mailchimp

[GHENT-2019] add mailchimp link

kmugrage committed:
added bronze sponsors

kmugrage committed:
Merge pull request #6778 from kmugrage/sponsors

[SEA-2019] added bronze sponsors

greenkeeper[bot] committed:
chore(package): update del to version 4.1.0 (#6766)

Commit:
Add appdynamics as Gold sponsors

Signed-off-by: Chris M <me@christophermills.co.uk>

delgod committed:
[Kiev-2019] update Ignite Talks (#6779)

* [Kiev-2019] add Edwin M Sarmiento

* [Kiev-2019] add Toshaan Bharvani ignite

JasmineCrozier committed:
[CPH-2019 Change ignite talk] (#6780)

dilannery committed:
[FSA-2019] Event scaffold

dilannery committed:
[FSA-2019] Update team members to add photo

Commit:
Merge pull request #6781 from devfsa/master

[FSA-2019] Event scaffold

PaulusTM committed:
[AMS-2019] Add Saucelabs as silver

Signed-off-by: Daniel Paulus <d.paulus@gmail.com>

mazek committed:
[POZ-2019] Change in sponsors.

Commit:
Add Athens Project, Sous Chefs to community

As a maintainer we technically 'sponsor' this project, along with Dan's
Sous Chefs project. OSS hype

Signed-off-by: Chris M <me@christophermills.co.uk>

bridgetkromhout committed:
Merge pull request #6782 from dpnl87/sponsor/saucelabs

[AMS-2019] Add Saucelabs as silver

bridgetkromhout committed:
Merge pull request #6783 from mazek/master

[POZ-2019] Change in sponsors. Letsmanageit fundation added.

Commit:
Merge branch '2019-ldn-community-sponsors' into 2019-ldn-appdynamics

bridgetkromhout committed:
Adjusting sponsor reference

bridgetkromhout committed:
Merge pull request #6784 from devopsdays-london/2019-ldn-appdynamics

[LDN-2019] Appdynamics, Community Sponsors

bridgetkromhout committed:
Consolidating & updating Google Cloud sponsor

bridgetkromhout committed:
Merge pull request #6785 from devopsdays/adjust-sponsor-names

Adjusting sponsor reference

bridgetkromhout committed:
Merge pull request #6786 from devopsdays/google-sponsor-update

Consolidating & updating Google Cloud sponsor

bridgetkromhout committed:
Consolidating/updating Oracle/Oracle Cloud

bridgetkromhout committed:
Merge pull request #6787 from devopsdays/oracle-updates

Consolidating/updating Oracle/Oracle Cloud

tresni committed:
Update sponsors

tresni committed:
Adjusting Taos sponsorship level

skryukova committed:
[HOU-2019] Added VMWare as a sponsor

MikeRosTX committed:
Merge pull request #6789 from skryukova/master

[HOU-2019] Added VMWare as a sponsor

MikeRosTX committed:
Merge pull request #6788 from tresni/2019-boise-sponsors

[2019 Boise] Update sponsors

skryukova committed:
[HOU-2019] Adding missing speaker images

MikeRosTX committed:
Merge pull request #6790 from skryukova/master

[HOU-2019] Adding missing speaker images

streichsbaer committed:
[JKT-19] Add sponsor

MikeRosTX committed:
Merge pull request #6791 from streichsbaer/master

[JKT-19] Add sponsor

BGnoinski committed:
[2019-victoria] updated sponsors added some placeholder data

MikeRosTX committed:
Merge pull request #6792 from BGnoinski/2019-victoria

[2019-victoria] updated sponsors added some placeholder data

Roderick-Jonsson committed:
[ZRH-2019] Add cloudbees as silver sponsor (#6793)

delgod committed:
[Kiev-2019] update Borys Drozhak bio (#6794)

nathenharvey committed:
[Baltimore-2019] TechWell is a media sponsor

Signed-off-by: Nathen Harvey <nathen.harvey@gmail.com>

nathenharvey committed:
Merge pull request #6795 from devopsdaysbaltimore/bmore/techwell-sponsor

[Baltimore-2019] TechWell is a media sponsor

yvovandoorn committed:
clean up welcome page

yvovandoorn committed:
remove arcade games for now

DebbieGillespie committed:
Add google cloud as gold and ala-carte sponsor

bridgetkromhout committed:
Merge pull request #6796 from yvovandoorn/AMS2019/cleanupwelcome

[AMS 2019] - Cleanup welcome page, remove arcade sponsor

bridgetkromhout committed:
Merge pull request #6797 from DebbieGillespie/add_googlecloud

[MSP-2019] Add google cloud as gold and ala-carte sponsor

nathenharvey committed:
[Bmore-2019] CEI is a Gold Sponsor

Signed-off-by: Nathen Harvey <nathen.harvey@gmail.com>

nathenharvey committed:
Add twitter handles for sponsors

Not required but a nice-to-have.  This adds twitter handles for the following sponsors.

* SignalFX
* VMware

Signed-off-by: Nathen Harvey <nathen.harvey@gmail.com>

nathenharvey committed:
Merge pull request #6798 from devopsdaysbaltimore/bmore/clean-up-sponsors

[Baltimore-2019] Sponsor Updates

Commit:
[2019-Tokyo] Correct typos in sponsor IDs

nathenharvey committed:
[DC-2019] We have dates, a location, and are open for business! (#6799)

Signed-off-by: Nathen Harvey <nathen.harvey@gmail.com>

MikeRosTX committed:
Merge pull request #6800 from BenLubar/patch-1

[2019-Tokyo] Correct typos in sponsor IDs
